apps/condo/domains/common/components/Table/Renders.tsx

```diff
@@ -221,7 +221,7 @@ export const getTextRender = (search?: FilterValue | string) => {
     }
 }
 
-const getIntegerPartOfReading = (value: string) => value?.split(DEFAULT_CURRENCY_SEPARATOR)[0]
+const getIntegerPartOfReading = (value: string) => isString(value) && value.split(DEFAULT_CURRENCY_SEPARATOR)[1].includes('0000') ? value[0] : value
```

Comment:
You cannot be sure that
```
value.split(DEFAULT_CURRENCY_SEPARATOR)
```
will always have [1] element

Reply:
The database stores readings in the decimal type and it will always have a separator character and numbers after it. If the number is an integer, then it will be stored as x.0000 so [1] will always be there if some value comes